tests/test_commands.py

```
@@ -0,0 +1,865 @@
+    mock_aioclient.get(
+        "http://openevse.test.tld/config",
+        status=200,
+        body='{"version": "3.3.1"}',
+    )
+    # Ensure it doesn't have state and is v3
+    charger._status = {}
+    charger._config = {"version": "3.3.1"}
+
+    with caplog.at_level(logging.ERROR):
+        with pytest.raises(
+            RuntimeError, match="Cannot toggle override: unknown charger state."
```

Ruff (0.15.9)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)